pkg/cmd/server/api/helpers.go

```diff
@@ -241,6 +241,14 @@ func GetMasterFileReferences(config *MasterConfig) []*string {
 
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.CertFile)
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.KeyFile)
+
+		for _, key := range []string{"cluster-signing-cert-file", "cluster-signing-key-file"} {
```

Comment:
I'm going to treat everything that ends in `-dir` and `-file` as "capable of being relativized".  I can't think how it wouldn't be backwards compatible.